frontend/public/index.html

```diff
@@ -24,12 +24,18 @@
       work correctly both with client-side routing and a non-root public URL.
       Learn how to configure a non-root public URL by running `npm run build`.
     -->
-    <title>React App</title>
-  </head>
-  <body>
-    <noscript>You need to enable JavaScript to run this app.</noscript>
-    <div id="root"></div>
-    <!--
+		<title>React App</title>
+		<!-- TODO: See if some of theses imports from quantum can be tossed (Some will be needed such as the bundle.js) -->
+		<link rel="shortcut icon" href="favicon.ico" />
+		<link rel="stylesheet" href="quantum/css/base.css" />
+		<script src="quantum/js/libs/numeric.js"></script>
```

Comment:
We can import this package directly from NPM and get ride of the lib in our repo if we import it like this
`<script src="https://unpkg.com/numeric@1.2.6/numeric-1.2.6.js"></script>`

Reply:
That makes it so that you can never work with this project without internet does it not?

We don't have to refactor this. I do want to refactor it eventually but right now qcsimulator works as is. It definitely needs reconsideration or cleanup in the future.

Reply:
I do think its a fair assumption that since there is a requirement for computation to be done in the backend and for the system to have user management it would always be in the cloud. We will refactor this another time to try and get this merged faster.